File: java/program/Publisher.java
class Publisher {
    // atribut-atribut private
    private String name;
    private String address;

    // constructor dengan parameter
    Publisher(String name, String address) {
        this.name = name;
        this.address = address;
    }

    // getter dan setter untuk name
    public void setName(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    // getter dan setter untuk address
    public void setAddress(String address) {
        this.address = address;
    }

    public String getAddress() {
        return address;
    }
}

File: cpp/program/Publisher.cpp
#pragma once
#include <bits/stdc++.h>

using namespace std;

// kelas publisher
class Publisher
{
private:
    // atribut-atribut private
    string name;
    string address;

public:
    // konstruktor kosong
    Publisher()
    {
    }

    // konstruktor dengan parameter untuk mengisi atribut
    Publisher(string name, string address)
    {
        this->name = name;
        this->address = address;
    }

    // setter name
    void setName(string name)
    {
        this->name = name;
    }

    // getter name
    string getName()
    {
        return name;
    }

    // setter address
    void setAddress(int address)
    {
        this->address = address;
    }

    // getter address
    string getAddress()
    {
        return address;
    }

    ~Publisher()
    {
    }
};